### ISSUE TITLE
[1.15.2] ElementText have no width so that other elements will cover it

### ISSUE THREAD

**Issue by SihenZhang** [McJty/TheOneProbe#411]
**Minecraft Version**: 1.15.2
**Forge Version**: 31.2.0
**TOP Version**: 2.0.4-5
**Description**:

I have made a test mod to ensure the width of ElementText is 0. Here is the code.
```java
public class TheOneProbeIntegration implements IProbeInfoProvider, Function<ITheOneProbe, Void> {
    private static final List<ItemStack> stackList = new ArrayList<>(4);

    static {
        stackList.add(new ItemStack(Items.OAK_LOG));
        stackList.add(new ItemStack(Items.STONE));
        stackList.add(new ItemStack(Items.DIAMOND));
    }

    @Override
    public Void apply(ITheOneProbe theOneProbe) {
        theOneProbe.registerProvider(this);
        return null;
    }

    @Override
    public String getID() {
        return Test.MOD_ID + ":test";
    }

    @Override
    public void addProbeInfo(ProbeMode mode, IProbeInfo probeInfo, PlayerEntity player, World world, BlockState blockState, IProbeHitData data) {
        Block block = blockState.getBlock();
        if (block instanceof TestBlock) {
            IProbeInfo stacks = probeInfo.vertical(probeInfo.defaultLayoutStyle());
            IProbeInfo stacksHorizontal = null;
            int cnt = 0;
            for (ItemStack stack : stackList) {
                if (cnt % 2 == 0) {
                    stacksHorizontal = stacks.horizontal(probeInfo.defaultLayoutStyle());
                }
                stacksHorizontal.horizontal(probeInfo.defaultLayoutStyle().alignment(ElementAlignment.ALIGN_CENTER))
                        .item(stack)
                        .text(stack.getDisplayName());
                cnt++;
            }
        }
    }
}
```
And in the debug mode, the evaluation value of the width is 0. 
![](https://i.imgur.com/pS4bzGL.png)
This will make text elements covered by the other children elements.
![](https://i.imgur.com/cEexFoR.png)

I also tried to use other methods to make it render correctly, like set a spacing for element using `FontRenderer#getStringWidth` method(which is also what TOP do to get the width of ElementText). But `addProbeInfo` method is called on server side, so TOP will show error.

**Comment by McJty** [comment]
I will fix that

**Comment by SihenZhang** [comment]
Seems that is has been fixed in the latest version.
https://github.com/McJtyMods/TheOneProbe/commit/21c24c17446863027949cd97a36776a00bd83f9a#diff-2163a51873e81962a5c57369c8467dc0
But TOP on the maven doesn't update yet.

**Comment by McJty** [comment]
Ah right. I forgot I also applied that fix to the 1.15 branch. I just updated the maven.
